File: dehaze-java/src/main/java/com/pei/dehaze/model/query/DictPageQuery.java
package com.pei.dehaze.model.query;


import com.pei.dehaze.common.base.BasePageQuery;
import io.swagger.v3.oas.annotations.media.Schema;
import lombok.Data;
import lombok.EqualsAndHashCode;

@Schema(description ="字典数据项分页查询对象")
@Data
@EqualsAndHashCode(callSuper = true)
public class DictPageQuery extends BasePageQuery {

    @Schema(description="关键字(字典项名称)")
    private String keywords;

    @Schema(description="字典类型编码")
    private String typeCode;
}

File: dehaze-go/model/query/dict_page_query.go
package query

// DictPageQuery 字典数据项分页查询对象
type DictPageQuery struct {
	// 关键字(字典项名称)
	Keywords string `json:"keywords"`
	// 字典类型编码
	TypeCode string `json:"typeCode"`
	// 页码
	PageNum int `json:"pageNum"`
	// 每页条数
	PageSize int `json:"pageSize"`
}